pkg/cmd/roachtest/tests/failure_injection.go

```diff
@@ -54,8 +57,29 @@ type failureSmokeTest struct {
 	workload func(ctx context.Context, c cluster.Cluster, args ...string) error
 }
 
+// annotateGrafana adds an annotation to both the internal and centralized Grafana
+// instance.
+func (t *failureSmokeTest) annotateGrafana(c cluster.Cluster, rt test.Test, text string) error {
+	errGroup := rt.NewErrorGroup()
+	errGroup.Go(func(ctx context.Context, l *logger.Logger) error {
```

Comment:
Kinda redundant... a helper to the rescue? E.g., `c.AddGrafanaAnnotation(..., all)`.